packages/autocomplete-core/src/createStore.ts

```diff
@@ -35,5 +35,6 @@ export function createStore<TItem extends BaseItem>(
 
       onStoreStateChange({ state, prevState });
     },
+    shouldSkipSearch: false,
```

Comment:
This naming makes more sense to me:

```suggestion
    shouldSkipPendingUpdate: false,
```

Reply:
Addressed in d663200.